api/api_sources/sources/server/modules/watercraftObservation/waterBody/waterBody.route.ts

```diff
@@ -25,7 +24,7 @@ export class WaterBodyRouteController extends ResourceRouteController<WaterBodyC
         return this.sharedInstance() as WaterBodyRouteController;
     }
 
-    async all(req: any, data?: any): Promise<[number, any]> {
+    /* async all(req: any, data?: any): Promise<[number, any]> {
```

Comment:
Is this commented out for any reason that we might use it later?

Reply:
Resolved